operations/commit_queue.go

```diff
@@ -362,11 +362,11 @@ func backport() cli.Command {
 				},
 				cli.StringFlag{
 					Name:  joinFlagNames(existingPatchFlag, "e"),
-					Usage: "existing commit queue patch",
+					Usage: "existing commit queue patch from another repository",
```

Comment:
I think this isn't likely to be a different repository but a different _project_ (or a different branch) bc backports happen to different branches on the same repo

Reply:
Oh! That was my misunderstanding, let me rephrase that